### ISSUE TITLE
Ornate Spyglass does not increase object/npc vision range

### ISSUE THREAD

**Issue by Inomares** [elysium-project/bug-tracker#655]
Ornate Sypglass (5507) is supposed to increase the range at which you can see distant gameobjects and NPCs, around the area you're zooming in on. I'm not sure by how much, but in vanilla it was apparently significant enough that players used it for scouting herb/mining nodes as it would apparently increase their vision range to more or less the entire minimap around the area you're looking at.

Vision range was not increased in battlegrounds, only in outdoor world.

Sources:
>"Increase the range on the mini-map that shows materials (mines/herbs/treasure)" - [WoWWiki ca 2006](http://wowwiki.wikia.com/wiki/Ornate_Spyglass?oldid=289728)
As well as a number of comments from https://web.archive.org/web/20070101040943/http://www.thottbot.com:80/?i=5420 - many pointing out that the increased vision range only worked outside of BGs. Please note that while the capture is from just after TBC was released, almost all of the comments are from vanilla.

**Comment by Thenari** [comment]
yeah i tested that a long time ago let me double check. sorry late addressing this, it wasnt super vital at the time!

**Comment by Thenari** [comment]
![image](https://user-images.githubusercontent.com/38762769/40104812-4a46f91e-58bf-11e8-81ef-738c46f1e9e4.png)

**Comment by Thenari** [comment]
![image](https://user-images.githubusercontent.com/38762769/40104870-689d80f4-58bf-11e8-8a02-3fd6259fe042.png)

**Comment by Thenari** [comment]
![image](https://user-images.githubusercontent.com/38762769/40104891-71ab089c-58bf-11e8-8dbb-2d0e51c9465d.png)

**Comment by Thenari** [comment]
isnt working still
